source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -768,7 +768,7 @@ namespace
             outputType = attr.getType();
         }
 
-        NodePtr destNode = dest->addNode(nodedef.getNode(), node.getName(), numOutputs > 1 ? "multioutput" : outputType);
+        NodePtr destNode = dest->addNode(nodedef.getNamespacedNode(), node.getName(), numOutputs > 1 ? "multioutput" : outputType);
```

Comment:
This was the main issue. We used to only use the "node" identifier to create a node on write e.g. "colorcorrect".
Since colorcorrect is namespaced, we need to create a "adsk:colorcorrect" node here.